### ISSUE TITLE
iat verification

### ISSUE THREAD

**Issue by omassi77** [Code-Hex/firebase-auth-cloudflare-workers#7]
An error occurs when the value of iat and the value of currentTimestamp are equal. However, on very fast networks, these values can be the same.
What do you think the code below can be changed to `if (currentTimestamp < payload.iat)` ?
https://github.com/Code-Hex/firebase-auth-cloudflare-workers/blob/a2d752f3320291c710afe0ebb5f3d67d79c10ff0/src/jwt-decoder.ts#L99

**Comment by nmemoto** [comment]
I have the same problems.
The error output I have seen is below.
```json
{
  message: 'failed to verify the requested firebase token',
  err: FirebaseAuthError: Decoding Firebase ID token failed. Make sure you passed the entire string JWT which represents an ID token. See https://firebase.google.com/docs/auth/admin/verify-id-tokens for details on how to retrieve an ID token. err: Error: Incorrect "iat" claim must be a older than "1678464161" (iat: "1678464161")
・・・・
    errorInfo: {
      code: 'auth/argument-error',
      message: 'Decoding Firebase ID token failed. Make sure you passed the entire string JWT which represents an ID token. See https://firebase.google.com/docs/auth/admin/verify-id-tokens for details on how to retrieve an ID token. err: Error: Incorrect "iat" claim must be a older than "1678464161" (iat: "1678464161")'
    },
    codePrefix: 'auth'
  }
}
```

**Comment by Code-Hex** [comment]
@omassi77 @nmemoto I'm sorry too late my reply 🙏 
I just published this fix as the latest version (v1.0.1) https://github.com/Code-Hex/firebase-auth-cloudflare-workers/releases/tag/v1.0.1

Thanks!!
